runtime-common/stdlib/msgpack/packer.h

```diff
@@ -64,6 +64,7 @@ class packer : private vk::not_copyable {
   void append_buffer(const char* buf, size_t len) noexcept;
 
   Stream& stream_;
+  bool has_error = false;
```

Comment:
Maybe add here {false}?) :smiley:. By the way `bool a{}` has same value as `bool a{false}`. Integrals types are initialised with null by default